### PR TITLE
Swedish translation added

### PR DESCRIPTION
**Pull request by hamalmar**

### CODE REVIEW

File: app/loc/lang-sv.json [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
     "STR_LOGOUT": "Logga ut",
-    "STR_MTG_LOGIN": "Logga in",
-    "STR_MTG_LOGIN_PROGRESS": "Sökar sammanträdet",
+    "STR_MTG_LOGIN": "Logga in till sammanträdet",
+    "STR_MTG_LOGIN_PROGRESS": "Söker sammanträdet",
     "STR_MTG_EXIT_PROGRESS": "Lämnar sammanträdet",
-    "STR_MTG_LOGOUT": "Lämna sammanträdet",
-    "STR_MTG_LOGOUT_CONFIRM": "Är du säker du vill stänga sammanträdet?",
+    "STR_MTG_LOGOUT": "Logga ut ur sammanträdet",
+    "STR_MTG_LOGOUT_CONFIRM": "Är du säker på att du vill lämna sammanträdet?",
```
Comment (tkenakka) [inline app/loc/lang-sv.json:11]: OK to merge but should you  double-check this from customer:
is it ok to say "stänga" here when leaving the view, can user be confused if "stänga" means closing the view or setting the meeting status "closed"?

File: app/loc/lang-sv.json [hunk 2]
```diff
@@ -82,35 +82,35 @@
...
     "STR_MOTION": "Motion",
-    "STR_MOTIONS_OPEN_APP": "Motionapplikation",
+    "STR_MOTIONS_OPEN_APP": "Motionsapplikationen",
     "STR_BUDGET_MOTION": "Budgetmotion",
-    "STR_SUPPORTED": "Understödet",
+    "STR_SUPPORTED": "Understödd",
     "STR_VOTINGS": "Omröstningar",
-    "STR_MTG_STATUS_PUBLISHED_PROPOSALS": "@:STR_PUBLIC_PROPOSALS",
-    "STR_PROP_PUB_COUNT": "*Antal av publicerad förslag",
+    "STR_MTG_STATUS_PUBLISHED_PROPOSALS": "Publicerade förslag",
```
Comment (tkenakka) [inline app/loc/lang-sv.json:99]: Ok to have like this, but notice the two strings have an identical translation.
Original `"STR_MTG_STATUS_PUBLISHED_PROPOSALS": "@:STR_PUBLIC_PROPOSALS"`
is correct as well: it's a reference to reuse the value from string `STR_PUBLIC_PROPOSALS`
Its up to which design you want: duplicate strings or reusing.

File: app/loc/lang-sv.json [hunk 3]
```diff
@@ -165,26 +165,27 @@
...
+    "STR_HELP_SIGNLIST": "I listan ser du alla dina dokument som kan undertecknas. Organets protokoll och tjänsteinnehavarbeslut är efter varandra som egna grupper. De behandlade dokumenten finns under en egen flik. Du kan öppna ett dokumentet genom att klicka på en knapp i raden.",
+    "STR_HELP_SIDEMENU": "Du kan ändra på sidmenyns bredd med hjälp av den här knappen.",
+    "STR_HELP_MENU_HDR": "Funktioner för vyns layout",
+    "STR_HELP_VIEWMODE": "Innehållet i vyn visas upp i sidmenyn och i två olika spalter. Du kan välja visningstyp för spalterna antingen bredvid eller ovanpå varandra.",
+    "STR_HELP_MTG_STATES_HDR": "Viktigaste statusar för sammanträdet",
     "STR_HELP_MTG_STATES_BODY": "Ehdotuksia voi julkaista kaikissa kokouksen tiloissa. Aloitteita voi jättää sähköisesti vain Kaupunginvaltuuston kokouksissa.",
```
Comment (tkenakka) [inline app/loc/lang-sv.json:188]: This is still unstranslated, could you ask what they want for this help dialog string?

**Comment by tkenakka** [comment]
LGTM with comments
smoke test ok, didn't check every string.

File: app/loc/lang-sv.json (hunk 1 above)
Comment (hamalmar) [reply app/loc/lang-sv.json:11]: The translation is now "Är du säker på att du vill lämna sammanträdet?" "stänga" is not used.

File: app/loc/lang-sv.json (hunk 3 above)
Comment (hamalmar) [reply app/loc/lang-sv.json:188]: Translation added in next version.

**Pull request closed without merging**